### ISSUE TITLE
It is not required to log into your youtube account :/

### ISSUE THREAD

**Issue by idkwhattoputhere121** [Louber1/YouTubeSubBot#2]
It is only required to log into the accounts of the 3 websites, logging into your youtube account is pointless

**Comment by Louber1** [comment]
No, because the Websites want you to subscribe to other Channels. And this is only possible if you're logged into your YouTube account ;)

**Comment by idkwhattoputhere121** [comment]
Ah, okay
